{
z_P0: [0.025, 0,   0,     # z
       0,     1.0, 0,     # z_dot
       0,     0,   0.01], # z_accel_bias
z_P0_flying: [0.25, 0, 0,
              0,    1, 0,
              0,    0, 0.01],
z_Q: [0.03, 0.0001],
z_R0: [0.0016],
z_R_flying: [0.0016],
z_x0: [-0.1, 0, 0.],
z_beta: [1.0, 1.0, 0.50]
}
